### PR TITLE
iOS - Issue #121 - Add Reminder Time row in Settings

### PR DESCRIPTION
**Pull request by jancy123**
Issue #121 

## Added the Reminder Time Row in settings
* Added the two labels 

Screenshot:
Simulator Screenshot - iPhone 15 - 2024-02-01 at 11 31 22

### CODE REVIEW

**Comment by jancy123** [comment]
@clc80 @devanshimodha This PR is ready to be reviewed. Thanks!

File: coding-projects/ios/TaskTracker/TaskTracker/Settings/SettingsView.swift [hunk 1]
```diff
@@ -71,6 +71,16 @@ private struct DaysView: View {
     }
 }
 
+private struct ReminderTimeView: View {
+    var body: some View {
+        HStack{
+            Text("REMINDER TIME")
```
[the remark this answers is not in the document]
Comment (jancy123) [reply coding-projects/ios/TaskTracker/TaskTracker/Settings/SettingsView.swift:77]: The wireframe had it uppercase. I have changed it to be capitalized. Here's the screenshot. If that looks better, I shall push the change.

Simulator Screenshot - iPhone 15 - 2024-02-01 at 11 31 22

[the remark this answers is not in the document]
Comment (clc80) [reply coding-projects/ios/TaskTracker/TaskTracker/Settings/SettingsView.swift:77]: > The wireframe had it uppercase. I have changed it to be capitalized. Here's the screenshot. If that looks better, I shall push the change.

Yes, that looks much better. The wireframes are suggestions not to be followed exactly.

**Review by clc80 — APPROVED** [review]
I will go ahead and approve your changes. You do have some merge conflicts that will need to be resolved before we can merge your PR. If you need help with the merge conflict feel free to ping either me or Devanshi in the open-source-and-personal-projects slack channel

**Review by devanshimodha — APPROVED** [review]

**Pull request merged**